src/renderer/store/modules/Note.js

```diff
@@ -53,16 +49,30 @@ const actions = {
         store.commit('SELECT_LOADING', true);
         store.commit('LOAD_NOTES', []);
 
-        octokit.authenticate({
-          type: 'token',
-          token: store.rootState.Settings.settings.githubPersonalAccessToken,
-        });
+        if (store.rootState.Settings.settings.githubEnterpriseUrl) {
+          octokit = Octokit({
+            baseUrl: store.rootState.Settings.settings.githubEnterpriseUrl,
+            userAgent: 'code-notes v1.2.3',
+            auth: store.rootState.Settings.settings.githubPersonalAccessToken,
+            mediaType: {
+              format: 'application/vnd.github.v3+json',
+            },
+          });
+        } else {
+          octokit = Octokit({
+            userAgent: 'code-notes v1.2.3',
+            auth: store.rootState.Settings.settings.githubPersonalAccessToken,
+            mediaType: {
+              format: 'application/vnd.github.v3+json',
+            },
+          });
```

Comment:
This code could be simplified if you set the octokit object before and then just add a `baseUrl` property only if `store.rootState.Settings.settings.githubEnterpriseUrl` is present.

Reply:
Yes I agree, this is due to my noob skills in java (python/java dev here), will def clean up as per your suggestion.